static/docs/commands-reference/status.md

```diff
@@ -118,14 +123,14 @@ cache. For the typical process to update workspaces, see
 ```dvc
 $ dvc status
 
-  bar.dvc
-          outs
-                  changed:  bar
-          deps
-                  changed:  foo
-  foo.dvc
-          outs
-                  changed:  foo
+bar.dvc:
+        changed deps:
+                changed:      bar
```

Comment:
This is an old status. Now we provide more info. See https://github.com/iterative/dvc.org/pull/416/files#r293507445

Reply:
@efiop  so shall I update this changed with any of these random statuses or do you want any particular one?

Reply:
@sanidhyamangal having multiple would be great. Thanks! :slightly_smiling_face:

Reply:
@efiop  okay cool, I will update them.